axiom/scripts/axiomatic.py

```diff
@@ -121,11 +121,15 @@ def parseOptions(self, args):
             # If a reactor is being selected, it must be done before the store
             # is opened, since that may execute arbitrary application code
             # which may in turn install the default reactor.
-            if "--reactor" in args:
-                reactorIndex = args.index("--reactor")
-                shortName = args[reactorIndex + 1]
-                del args[reactorIndex:reactorIndex + 2]
-                self.opt_reactor(shortName)
+            for index, arg in enumerate(args):
+                if arg.startswith("--reactor"):
```

Comment:
Yes, I tried that approach first, for the reasons you mention above, but thought the likelihood of a --reactor-foo argument being added to twisted or axiom at some point seemed slight.  I can definitely change this if you prefer.